diagnostic.md

```diff
@@ -36,16 +36,17 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+It is a relative path because an absolute path would start with the name of the current user's home directory.
```

Comment:
This would be an absolute path. Reason being that it starts at the root (aka. `/Users/blah_blah/Desktop`)